### DISCUSSION TITLE
1️⃣ Share your profiles!

### DISCUSSION THREAD

**Discussion by crazy4pi314** [crazy4pi314/scipy-vscode-tutorial#17] [Show and tell]
In the first chapter of the workshop, you will have created and exported a profile of your custom UI config. Add a link to the gist in the comments below so other participants can check it out!

**Comment by rekcut-nod** [comment]
https://vscode.dev/profile/github/c49a14a5b1d49d1af159b77e2ec36fc3

**Reply by rhiannong123** [reply]
https://vscode.dev/profile/github/0cbf400a62ef3183a69150323a2bcb86

**Comment by anzelpwj** [comment]
https://vscode.dev/profile/github/03b0c0642f74f37bb0dd238755c8fed4

**Comment by Kab00mKap0w** [comment]
https://vscode.dev/profile/github/b478a08f2524f25d0d410986ba674065

**Comment by bjcubsfan** [comment]
https://vscode.dev/profile/github/d3a95d4c82e66976b3f656ee0a2b8ffb

**Comment by Jeffotter** [comment]
https://vscode.dev/profile/github/f6bdbe4f892f3e0046ec5f83a02170ea

**Comment by eni-awowale** [comment]
https://vscode.dev/profile/github/ce939077eac395266137130a5d980615
